resources/schema.py

```diff
@@ -452,6 +461,55 @@ def mutate_and_get_payload(cls, root, info, **kwargs):
         return DeleteHarborMutation()
 
 
+class PierInput(AbstractAreaSectionInput):
+    harbor_id = graphene.ID()
+    suitable_boat_types = graphene.List(graphene.String)
```

Comment:
list of IDs?
```python
suitable_boat_types = graphene.List(graphene.ID)
```

Reply:
The `BoatType` query uses plain IDs, not GQL Global IDs

```graphql
query BT {
  boatTypes {
    id
  }
}
```

```json
{
  "data": {
    "boatTypes": [
      {
        "id": "1"
      },
      {
        "id": "2"
      },
      {
        "id": "3"
      },
      {
        "id": "4"
      },
      {
        "id": "5"
      },
      {
        "id": "6"
      },
      {
        "id": "7"
      },
      {
        "id": "8"
      }
    ]
  }
}
```

Reply:
yes, plain ID can be a string or an integer in graphene

Reply:
I think this might be valid list `["1", 2, "3"]` 🤔

Reply:
Just FTR, it is a [valid id](https://docs.graphene-python.org/en/latest/types/scalars/).
```
graphene.ID represents a unique identifier, often used to refetch an
object or as key for a cache. The ID type appears in a JSON
response as a String; however, it is not intended to be
human-readable. When expected as an input type, any string
(such as “4”) or integer (such as 4) input value will be accepted as an ID.
```

Reply:
yeah, that's what I meant. 🙂